workflow/controller/operator_test.go

```diff
@@ -5351,6 +5351,392 @@ func TestPodFailureWithContainerWaitingState(t *testing.T) {
 	assert.Contains(t, msg, "Pod failed before")
 }
 
+var podWithWaitContainerOOM = `
+apiVersion: v1
```

Comment:
look good, but can I ask you to make this example minimal - it's very long and will be hard to maintain